cms/djangoapps/contentstore/views/course.py

```diff
@@ -999,7 +999,6 @@ def update_tab(tabs, tab_type, tab_enabled):
         """
         tab_panel = {
             "type": tab_type.name,
-            "name": tab_type.title,
         }
         has_tab = tab_panel in tabs
```

Comment:
The tabs logic is a little odd, but if you're comparing a dictionary against a `CourseTab` object, it will only check the `type` and not the `name` if there's only a `type`: https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/tabs.py#L139